src/index.js

```diff
@@ -80,19 +86,51 @@ export default function compile(services: Services, blocks: Blocks): GraphResult
       .forEach((topic) => {
         addSubscriber(name, topic);
       });
+
+    // LocalPublishers
+    resolvedBlockNames
+    .map(b => localPubs[b])
+    .filter(exists)
+    .forEach((topic) => {
+      addLocalPublisher(name, topic);
+    });
+
+    // LocalSubscribers
+    resolvedBlockNames
+      .map(b => localSubs[b])
+      .filter(exists)
+      .forEach((topic) => {
+        addLocalSubscriber(name, topic);
+      });
   });
 
   const pubTopics = Object.keys(topicToPubs);
   const subTopics = Object.keys(topicToSubs);
+  const localPubTopics = Object.keys(topicToLocalPubs);
+  const localSubTopics = Object.keys(topicToLocalSubs);
 
-  const edges = subTopics.reduce((l0, subTopic) => (
+  const pubSubEdges = subTopics.reduce((l0, subTopic) => (
```

Comment:
i like the idea, for consistency with the rest of the variable names, of keeping this named `edges`, then removing line 132 and just do the concatenation on line 136.